iot/manager/manager.go

```diff
@@ -138,21 +152,23 @@ func getRegistryIam(projectID string, region string, registryID string) (*cloudi
 		return nil, err
 	}
 
-	fmt.Println("Policy:")
+	fmt.Fprintf(w, "Policy:\n")
```

Comment:
Small comment, `Fprintln` is OK if you aren't printing any variables. Whatever you prefer.

Reply:
Good catch. Changed to using `Fprintln` for no variables prints